### PR TITLE
resolve conflict and done testing and it show the course list

### PR DESCRIPTION
**Pull request by ziqiang2344**

### CODE REVIEW

File: src/sample/classes/StudentController.java [hunk 1]
```diff
@@ -18,67 +21,67 @@
...
-  @FXML
-  private AnchorPane anchorid;
-  @FXML
-  private TextField textfield;
+public class StudentController implements Initializable{
+@FXML
+private Label label;
+@FXML
+private AnchorPane anchorid;
+@FXML
```
Comment (Michelle-Lohwt) [inline src/sample/classes/StudentController.java]: remove the space after that one ahhahaha

File: student1courseList.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+CPT111 - PRINCIPLES OF PROGRAMMING
```
Comment (Michelle-Lohwt) [inline student1courseList.txt]: remove the `student1courseList.txt`, it is created when sign up but is not necessary for the application until a real user use it

File: student1registeredCourse.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+CPT112 - DISCRETE STRUCTURES
```
Comment (Michelle-Lohwt) [inline student1registeredCourse.txt]: remove the `student1registeredCourse.txt`, it is created when signing up but is not necessary for the application until a real user use it

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Check the changes requested

File: student1registeredCourse.txt (hunk 3 above)
Comment (ziqiang2344) [reply student1registeredCourse.txt]: Done all the correction above

File: src/sample/classes/StudentController.java [hunk 4]
```diff
@@ -18,67 +21,67 @@
...
 
-  }
 
-  @FXML
-  public void confirmbutton(ActionEvent event) {
-    Stage stage = (Stage) anchorid.getScene().getWindow();
-    stage.close();
+  
+}
+
```
Comment (Michelle-Lohwt) [inline src/sample/classes/StudentController.java:84]: when the user click confirm can it show the `The pdf is created`, then ask the user to close the window?

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Improve suggestion

File: src/sample/classes/StudentController.java (hunk 4 above)
Comment (ziqiang2344) [reply src/sample/classes/StudentController.java:84]: i removed the confirm button as i think it's redundant to let users click twice just like when we upload files to Gdrive, it won't ask us to confirm again. 

For the 'The pdf is created' message I can try

**Comment by Michelle-Lohwt** [comment]
@ziqiang2344 boss, got conflict again

**Comment by Michelle-Lohwt** [comment]
@ziqiang2344 Got conflict, but mcm not very hard conflict

**Pull request closed without merging**